File: MORM.Java/src/MORM/Java/Models/Venda/Histrel.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "HISTREL")
public class Histrel extends CollectionItem
{
    @CampoAttribute(Campo = "ID_HISTREL", Tipo = CampoTipo.Key)
    public Integer Id_Histrel;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "TP_DOCUMENTO", Tipo = CampoTipo.Req)
    public Integer Tp_Documento;
    @CampoAttribute(Campo = "CD_HISTREL", Tipo = CampoTipo.Req)
    public Integer Cd_Histrel;
    @CampoAttribute(Campo = "DS_HISTREL", Tipo = CampoTipo.Req)
    public String Ds_Histrel;
    @CampoAttribute(Campo = "NR_PARCELAS", Tipo = CampoTipo.Req)
    public Integer Nr_Parcelas;
}

File: MORM.CSharp/MORM.CSharp/Models/Venda/Histrel.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("HISTREL")]
    public class Histrel : CollectionItem
    {
        [Campo("ID_HISTREL", CampoTipo.Key)]
        public int Id_Histrel { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("TP_DOCUMENTO", CampoTipo.Req)]
        public int Tp_Documento { get; set; }
        [Campo("CD_HISTREL", CampoTipo.Req)]
        public int Cd_Histrel { get; set; }
        [Campo("DS_HISTREL", CampoTipo.Req)]
        public string Ds_Histrel { get; set; }
        [Campo("NR_PARCELAS", CampoTipo.Req)]
        public int Nr_Parcelas { get; set; }
    }
}